frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -34,13 +34,25 @@ const TopContent = styled.div`
     display: flex;
     justify-content: space-between;
 `
+const MiddleContent = styled.div`
+    display: flex;
+    justify-content: flex-left;
+    gap: 40px;
+    white-space: nowrap;
+`
+
 const BottomContent = styled.div`
     display: flex;
     justify-content: space-between;
```

Comment:
Should there be the same justify-content for both MIddleContent and BottomContent?